catalog/dags/common/cloudwatch.py

```diff
@@ -71,3 +71,4 @@ def enable_or_disable_alarms(enable):
     )
     cw_wrapper = CloudWatchWrapper(cloudwatch.get_conn())
     cw_wrapper.enable_alarm_actions("ES Production CPU utilization above 50%", enable)
+    cw_wrapper.enable_alarm_actions("ES Production CPU utilization above 90%", enable)
```

Comment:
Sorry for the slight correction, this alarm is actually _slightly_ different (I'll make an issue to correct this in our infrastructure repository):
```suggestion
    cw_wrapper.enable_alarm_actions("ES Prod CPU utilization above 90%", enable)
```